docs/layouts/shortcodes/cli-installation-requirements.md

```diff
@@ -0,0 +1,7 @@
+You need to have:
+
+* [PHP 5.5.9+](https://www.php.net/manual/en/install.php) with the following extensions installed: `curl`, `json`, `mbstring`, `pcre`, and `phar`.
+* [Git](https://docs.github.com/en/get-started/quickstart/set-up-git).
+* A Bash-like shell.
+
+On Windows, to install PHP and Git you can use [chocolatey](https://docs.chocolatey.org/en-us/). The best way to get Bash is through [Windows Subsystem for Linux](https://msdn.microsoft.com/en-gb/commandline/wsl/about).
```

Comment:
:clipboard:  
```suggestion
On Windows, to install PHP and Git you can use [Chocolatey](https://docs.chocolatey.org/en-us/).
The best way to get Bash is through [Windows Subsystem for Linux](https://msdn.microsoft.com/en-gb/commandline/wsl/about).
```

Reply:
:lady_beetle: 
Chocolately recommends against organizations/businesses using their repository. Maybe we shouldn't be recommending it. I'm not sure installing PHP or Git is so difficult without it. :shrug: